lib/perl/Genome/VariantReporting/Command/Wrappers/EpitopeBindingPredictionFasta.pm

```diff
@@ -0,0 +1,78 @@
+package Genome::VariantReporting::Command::Wrappers::EpitopeBindingPredictionFasta;
+
+use strict;
+use warnings;
+use Genome;
+use File::Basename qw(dirname);
+use YAML;
+
+class Genome::VariantReporting::Command::Wrappers::EpitopeBindingPredictionFasta {
+    is => 'Command::V2',
+    has_input => [
+        model => {
+            is => "Genome::Model::SomaticVariation",
+        },
+        output_directory => {
+            is => 'Path',
+        },
+    ],
+    has_calculated_optional => [
+        build => {
+            calculate_from => [qw(model)],
+            calculate => q/$model->last_succeeded_build/,
+        },
+        translations_file => {
+            calculate_from => [qw(output_directory)],
+            calculate => q/File::Spec->join($output_directory, "resources.yaml")/,
+        },
+    ],
+};
+
+sub execute {
+    my $self = shift;
+    $self->generate_translations_file;
+    $self->run_reports;
+}
+
+
+sub generate_translations_file {
+    my $self = shift;
+    my %translations;
+    $translations{reference_fasta} = $self->build->tumor_build->reference_sequence_build->full_consensus_path("fa");
```

Comment:
Yeah, there is some overlap, mostly in the helper functions which are actually used in other places as well. Time for a ReportHelpers class of sorts!